### ISSUE TITLE
bottleneck conflict during install...

### ISSUE THREAD

**Issue by drelo** [faircloth-lab/phyluce#57]
I noticed your post [here](https://github.com/conda/conda/issues/2436) so I am aware of the issue but is this solved?

I am using Ubuntu 16.04, anaconda2 (fresh install, today) and tried to install phyluce today. 

```

→ conda install -c faircloth-lab phyluce      
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata .............
Solving package specifications .......

The following specifications were found to be in conflict:
  - bottleneck -> numpy 1.10*|1.11*|1.9*
  - bottleneck -> python 3.4*|3.5*
  - phyluce
Use "conda info <package>" to see the dependencies for each package.
```

Thanks for the feedback

**Comment by brantfaircloth** [comment]
I hear you.  This was introduced in Anaconda 3.  Been trying to track down exactly why this is happening and what is the best fix (e.g., see https://github.com/conda/conda/issues/2436).  The short term fix is to do one of the following:

* run `conda remove bottleneck`
* run `conda install phyluce`

or remove the entire `anaconda` install directory, re-install using the following version:

http://repo.continuum.io/archive/Anaconda2-2.5.0-Linux-x86_64.sh

After that, add the phyluce repository, then `conda install phyluce`.

Hope that helps a bit - and very sorry for the problems (it's not entirely my fault - and, working on a fix, albeit a bit slowly because of some other priorities).

-b

**Comment by brantfaircloth** [comment]
Yes - please keep this open.  I entirely agree it's annoying.

**Comment by drelo** [comment]
Thanks, I am kinda newbie to conda so I didn't know if removing bottleneck
would break things downstream. Thanks for your ultrafast reply.

El mar., 14 de jun. de 2016 a la(s) 17:16, Brant Faircloth <
notifications@github.com> escribió:

> I hear you. This was introduced in Anaconda 3. Been trying to track down
> exactly why this is happening and what is the best fix (e.g., see
> conda/conda#2436 <https://github.com/conda/conda/issues/2436>). The short
> term fix is to do one of the following:
>
>    - run conda remove bottleneck
>    - run conda install phyluce
>
> or remove the entire anaconda install directory, re-install using the
> following version:
>
> http://repo.continuum.io/archive/Anaconda2-2.5.0-Linux-x86_64.sh
>
> After that, add the phyluce repository, then conda install phyluce.
>
> Hope that helps a bit - and very sorry for the problems (it's not entirely
> my fault - and, working on a fix, albeit a bit slowly because of some other
> priorities).
>
> -b
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/faircloth-lab/phyluce/issues/57#issuecomment-226019027>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ2mis8ufNMlRlTmbKvS_3t2bu3bBqEPks5qLxpIgaJpZM4I1w_h>
> .
>
-- 

Postdoctoral Associate

Instituto de Cs Ambientales y Evolutivas
Universidad Austral de Chile
Campus Isla Teja s/n
Valdivia, Chile

**Comment by prehensilecode** [comment]
Still getting an error, after removing "bottleneck". 

```
$ conda --version
conda 4.3.25
$ python --version
Python 2.7.13 :: Anaconda custom (64-bit)
$ conda list | grep bottleneck
(no output)
$ conda install phyluce
Fetching package metadata ...........
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - phyluce -> biopython 1.63 -> numpy 1.7* -> mkl 10.3
  - pywavelets
Use "conda info <package>" to see the dependencies for each package.

```

**Comment by brantfaircloth** [comment]
you can also remove `navigator-updater` and `seaborn`.  hopefully that will clear the error.  i need
to write up an official work-around...

**Comment by prehensilecode** [comment]
Removing `navigator-updater` and `seaborn` would also remove `anaconda`:

`$ conda remove navigator-updater seaborn
Fetching package metadata ...........
Solving package specifications: .

Package plan for package removal in environment /home/someone/anaconda3:

The following packages will be REMOVED:

    anaconda:          4.4.0-np112py36_0
    navigator-updater: 0.1.0-py36_0
    seaborn:           0.7.1-py36_0`

**Comment by brantfaircloth** [comment]
ok, two problems here

1. in your email (which I don't see in the thread above, for whatever reason) it looks like you are working with `anaconda3` - that's not going to work with phyluce regardless - you need an anaconda with python 2.7.x.  

2. to try and work around the second problem (blocking packages), try:

    ```
    conda create --name phyluce python=2
    source activate phyluce
    conda install phyluce
    # do some work
    # when finished
    source deactivate
    ```
This creates a specific environment in which to run phyluce, without the other packages that appear to be causing conflicts.

**Comment by prehensilecode** [comment]
Sorry, I was working on the user's account, and they had installed anaconda3 replacing the anaconda2 from my first messages. As soon as I realized that, I deleted that comment in this thread, which is why you got the email but don't see it here.

Creating a new environment "phyluce" works, and `phyluce` installs with no problem.  Thanks much!

**Comment by brantfaircloth** [comment]
No worries - I figured something like that probably happened.  glad the 
new env helped.  I’ll add that to the documentation shortly.

-b

**Comment by brantfaircloth** [comment]
This is fixed in v1.6.2 and following the new installation instructions (using bioconda): http://phyluce.readthedocs.io/en/latest/installation.html
